### PR TITLE
Add GNS3.app v1.3.10

### PR DESCRIPTION
**Pull request by samdoran**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seems like this is a [walled build](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#unofficial-vendorless-and-walled-builds).

File: Casks/gns3.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  homepage 'http://www.gns3.com'
+  license :gpl
+
+  app 'GNS3.app'
+
+  uninstall :quit => 'net.gns3'
+
+  zap :delete => [
+                    '~/Library/Preferences/net.gns3.plist',
+                    "~/.config/gns3.net",
```
Comment (vitorgalvao) [inline Casks/gns3.rb]: Double quotes are reserved for cases where string interpolation is needed.

Comment (samdoran) [inline Casks/gns3.rb]: @vitorgalvao Sorry about that. I fixed it and pushed a commit with single quotes.

Comment (vitorgalvao) [inline Casks/gns3.rb]: [This is a bigger issue](https://github.com/caskroom/homebrew-cask/pull/14041#issuecomment-143797385). We cannot accept this cask here.

**Comment by samdoran** [comment]
@vitorgalvao I don't believe this is a walled build. The download link from their site _does_ take you to an account creation page, but the download link once you get past the login is a GitHub link. I understand if this toes the line and cannot be included.

**Comment by vitorgalvao** [comment]
It’s still a walled build. A user auditing this cask has no way to know that after the registration it’ll be a Github URL. [This cask was refused before](https://github.com/caskroom/homebrew-cask/pull/9099).

**Comment by samdoran** [comment]
@vitorgalvao Ok. Sorry for causing rework for you. Thank you for taking the time to explain.

**Comment by vitorgalvao** [comment]
No problem. Thank you anyway for the contribution (and for your past ones).

**Pull request closed without merging**